src/components/flow/actions/sendinteractivemsg/SendInteractiveMsgForm.tsx

```diff
@@ -140,12 +140,13 @@ export default class SendMsgForm extends React.Component<
     if (this.state.expression) {
       valid = true;
     }
-
     if (valid) {
       this.props.updateAction(stateToAction(this.props.nodeSettings, this.state));
       if (this.props.nodeSettings.originalNode.ghost) {
         this.props.resetNodeEditing();
-        this.props.updateRouter(stateToRouter(this.props.nodeSettings, this.state));
+        this.props.updateRouter(
+          stateToRouter(this.props.nodeSettings, this.props.assetStore, this.state)
```

Comment:
we are changing the signature of the function, I would prefer adding new params at the end.